tests/rule_based_profiler/test_profiler_run_anonymizer.py

```diff
@@ -0,0 +1,525 @@
+                        "expectation_type": "expect_column_values_to_match_regex",
+                    }
+                ],
+                "anonymized_name": "5a83f3728393d6519a197cffdccd50ff",
+                "anonymized_parameter_builders": [
+                    {
+                        "anonymized_batch_request": {
+                            "anonymized_batch_request_required_top_level_properties": {
+                                "anonymized_data_asset_name": "eac128c5824b698c22b441ada61022d4",
+                                "anonymized_data_connector_name": "123a3221fc4b65014d061cce4a71782e",
+                                "anonymized_datasource_name": "df78ebde1957385a02d8736cd2c9a6d9",
+                            },
+                            "batch_request_optional_top_level_keys": [
+                                "data_connector_query"
+                            ],
+                        },
+                        "anonymized_name": "9349ed253aba01f4ecf190af61018a11",
+                        "parent_class": "MetricMultiBatchParameterBuilder",
+                    }
+                ],
+            }
+        ],
+        "variable_count": 1,
+        "rule_count": 1,
+    }
+
+
+def test_anonymize_profiler_run_with_condition_in_expectation_configuration_builder(
+    profiler_run_anonymizer: ProfilerRunAnonymizer,
+    usage_stats_profiler_config: dict,
+) -> None:
+    rules: Dict[str, dict] = usage_stats_profiler_config["rules"]
+    expectation_configuration_builder: dict = rules["rule_1"][
+        "expectation_configuration_builders"
+    ][0]
+    expectation_configuration_builder["condition"] = "my_condition"
+
+    anonymized_result: dict = profiler_run_anonymizer.anonymize_profiler_run(
+        **usage_stats_profiler_config
+    )
+    assert anonymized_result == {
+        "anonymized_name": "5b6c98e19e21e77191fb071bb9e80070",
+        "anonymized_rules": [
+            {
+                "anonymized_domain_builder": {"parent_class": "TableDomainBuilder"},
+                "anonymized_expectation_configuration_builders": [
+                    {
+                        "anonymized_condition": "553b1c035d9b602798d64d23d63abd32",
+                        "parent_class": "DefaultExpectationConfigurationBuilder",
+                        "expectation_type": "expect_column_values_to_match_regex",
+                    }
+                ],
+                "anonymized_name": "5a83f3728393d6519a197cffdccd50ff",
+                "anonymized_parameter_builders": [
+                    {
+                        "anonymized_name": "9349ed253aba01f4ecf190af61018a11",
+                        "parent_class": "MetricMultiBatchParameterBuilder",
+                    }
+                ],
+            }
+        ],
+        "config_version": 1.0,
+        "rule_count": 1,
+        "variable_count": 1,
+    }
+
+
+def test_resolve_config_using_acceptable_arguments(
```

Comment:
Thank you for these tests. Though tests related to `resolve_config_using_acceptable_arguments` should go into the main RuleBasedProfilerConfig test file, not the anonymizer. Would you mind moving?